Until wxWidgets 3.2.x is fixed for older macOS, this is unavoidable.

--- configure	2024-04-16 02:20:29.000000000 +0800
+++ configure	2024-04-27 22:26:10.000000000 +0800
@@ -19636,7 +19636,7 @@
 
 
 
-    MIN_WX_VERSION="3.2.1" # Do not downgrade this. wx 3.2.0 and older have too many known and fixed bugs affecting FileZilla.
+    MIN_WX_VERSION="3.0.4" # Do not downgrade this. wx 3.0.3 and older have too many known and fixed bugs affecting FileZilla.
 
 
     if test x${WX_CONFIG_NAME+set} != xset ; then
